src/test/Components/Blueprints/Blueprints.test.js

```diff
@@ -50,37 +52,51 @@ describe('Blueprints', () => {
   });
   test('renders blueprint composes', async () => {
     renderWithReduxRouter('', {});
+    const idMatcher = blueprintIdWithComposes;
     const nameMatcher = (_, element) =>
-      element.getAttribute('name') === blueprintNameWithComposes;
+      element.getAttribute('name') === 'blueprints';
 
-    const blueprintRadioBtn = await screen.findByRole('radio', {
+    const radioButtons = await screen.findAllByRole('radio', {
       name: nameMatcher,
     });
-    await user.click(blueprintRadioBtn);
+    const elementById = radioButtons.find(
+      (button) => button.getAttribute('id') === idMatcher
+    );
+
+    await user.click(elementById);
     const table = await screen.findByTestId('images-table');
     const { findByText } = within(table);
     await findByText(blueprintNameWithComposes);
   });
   test('renders blueprint composes empty state', async () => {
     renderWithReduxRouter('', {});
+    const idMatcher = blueprintIdEmptyComposes;
     const nameMatcher = (_, element) =>
-      element.getAttribute('name') === blueprintNameEmptyComposes;
+      element.getAttribute('name') === 'blueprints';
 
-    const blueprintRadioBtn = await screen.findByRole('radio', {
+    const radioButtons = await screen.findAllByRole('radio', {
       name: nameMatcher,
     });
-    await user.click(blueprintRadioBtn);
+    const elementById = radioButtons.find(
+      (button) => button.getAttribute('id') === idMatcher
+    );
+    await user.click(elementById);
     expect(screen.queryByTestId('images-table')).not.toBeInTheDocument();
   });
   test('click build image button', async () => {
     renderWithReduxRouter('', {});
+    const idMatcher = blueprintIdWithComposes;
     const nameMatcher = (_, element) =>
-      element.getAttribute('name') === blueprintNameWithComposes;
+      element.getAttribute('name') === 'blueprints';
 
-    const blueprintRadioBtn = await screen.findByRole('radio', {
+    const radioButtons = await screen.findAllByRole('radio', {
       name: nameMatcher,
     });
-    await user.click(blueprintRadioBtn);
+    const elementById = radioButtons.find(
+      (button) => button.getAttribute('id') === idMatcher
+    );
+
+    await user.click(elementById);
```

Comment:
I was thinking that once we changed the radio buttons to clickable card, it would be easier to access the elements, though, it seems that PF still renders radio buttons. 🤔